doc/manual.rst

```diff
@@ -4529,8 +4529,10 @@ Example:
       echo "overflow!"
     except ValueError, IOError:
       echo "catch multiple exceptions!"
-    except:
-      echo "Unknown exception!"
+    except CatchableError:
+      echo "Catchable exception!"
+    except Defect:
+      echo "Defect!
```

Comment:
```suggestion
      echo "Defect!"
```

Reply:
thanks!

Reply:
You can add a ` :test: ` below the `.. code-block:: nim` line to compile the code,
so this bug can not happen again, see others in the same file. 🙂

Reply:
> You can add a :test: below the .. code-block:: nim line to compile the code,
so this bug can not happen again, see others in the same file. 🙂

You said it! Thanks!